cmd/postgres-operator/main.go

```diff
@@ -111,15 +110,16 @@ func main() {
 	// deprecation warnings when using an older version of a resource for backwards compatibility).
 	rest.SetDefaultWarningHandler(rest.NoWarnings{})
 
-	namespaces, err := k8s.GetWatchNamespace()
+	options, err := initManager(ctx)
 	assertNoError(err)
 
+	options.LeaderElection = true
+	options.LeaderElectionID = perconaRuntime.ElectionID
```

Comment:
Since we are setting these here, why don't we move them inside the initmanager?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1135/commits/0434c3997413cf07a63d163fbb371f4ca1a33714

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1135/commits/30a70f5b2d1f9c28b7b70b10e4fe70fc86af07de